src/main/java/com/tobmistaketracker/TobMistake.java

```diff
@@ -18,6 +18,7 @@ public enum TobMistake {
     DEATH_XARPUS("Xarpus Death", "death_xarpus.png", "I'm planking!"),
     DEATH_VERZIK("Verzik Death", "death_verzik.png", "I'm planking!"),
     MAIDEN_BLOOD("Maiden Blood", "maiden_blood.png", "I'm drowning in Maiden's blood!"),
+    MAIDEN_BLOOD_BLOB("Maiden Blood (Blob)", "maiden_blood_orb.png", "I'm gonna catch this blob!"),
```

Comment:
Hmm, I've never heard of it called a "blob" before. That usually means the inferno blobs. How do you feel about changing this to Blood Spawn? I've heard tomato if you prefer that, lol.